contracts/staking/ServiceStakingBase.sol

```diff
@@ -191,7 +281,6 @@ abstract contract ServiceStakingBase is IErrorsRegistries {
     /// @param eligibleServiceIds Service Ids eligible for rewards.
     /// @param eligibleServiceRewards Corresponding rewards for eligible service Ids.
     /// @param serviceIds All the staking service Ids.
-    /// @param serviceNonces Current service nonces.
```

Comment:
why this comment was removed?

Reply:
It was from the previous PR when this was considered obsolete, I've changed that now.